### DISCUSSION TITLE
modification of blocs

### DISCUSSION THREAD

**Discussion by clemaiz** [woocommerce/woocommerce#51688] [Checkout Flow]
Hello, how to modify payment and cart blocks on woocommerce? 
Thank you for your answer.

**Comment by mikejolley** [comment]
Hello,

I understand you're wanting to modify the payment areas and cart blocks in WooCommerce, but without more detail I cannot really point you in a specific direction, but we do have documentation regarding customisations you can make:

https://woocommerce.com/document/woocommerce-store-editing/customizing-cart-and-checkout/

See also: https://woocommerce.com/document/woocommerce-store-editing/customizing-cart-and-checkout/cart-block/

I would suggest posting questions (with a little more detail) on the plugin support forums to speak to other users and developers: https://wordpress.org/support/plugin/woocommerce/

Thank you
